relays/table_transform.ts

```diff
@@ -23,10 +24,10 @@ export default function(msg: Message): void|Message|Message[]|Promise<Message> {
 	let rounds = roundsArray(msg.data.stage.rounds);
 
 	let data = {
-		header: [NUMBER, NAME, TOP].concat(rounds),
-		data: msg.data.ranking.map(function(rank) {
+		header: [RANK, NUMBER, NAME, TOP].concat(rounds),
+		data: msg.data.ranking.map(function(rank, index) {

```

Comment:
This won't work for two reasons:
- `index` is zero-based
- some teams may share the same rank, if they have the exact same score

Because of this, the broadcasted message already contains a `rank` property.

You can opt to directly use that, but it could look like:
```
1. Some team, 100 points
1. Another team, 100 points
2. Yet another team, 80 points
```

It's probably best to leave it to the display system to render it in practice like e.g.:
```
1. Some team, 100 points
   Another team, 100 points
2. Yet another team, 80 points
```

(Reason for letting display system handle it, is because lists may span multiple 'pages', and a shared rank may split such a page, thus maybe needing to repeat the ranking in some cases.)